File: PRAK304-2310817310008-MUHAMMADRIZKIRAMADHAN.py
a = int(input(" "))

if (a >= 0) :
    if (a == 0) : print("Nol")
    elif (a > 0 and a < 10) : print("Satuan")
    elif (a > 10 and a < 20) : print("Belasan")
    elif (a == 10 or a >= 20 and a < 100) : print("Puluhan")
    else : print("Anda Menginput Melebihi Limit Bilangan")

File: PRAK304-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>

int main(){
    int a;
    printf(" ");
    scanf("%d", &a);

    if (a >= 0){
        if (a == 0){
        printf("Nol");
        }
        else if ( a > 0 && a < 10){
        printf("Satuan");
        }
        else if (a > 10 && a < 20){
        printf("Belasan");
        }
        else if (a == 10 || a >= 20 && a <100){
        printf("Puluhan");
        }
        else{
        printf("Anda Menginput Melebihi Limit Bilangan");
        }
    }
    return 0;
}